lib/broccoli/ember-app.js

```diff
@@ -679,23 +674,21 @@ class EmberApp {
     @return {Tree}
   */
   _filterAppTree() {
-    if (this._cachedFilterAppTree) {
-      return this._cachedFilterAppTree;
+    if (!this._cachedFilterAppTree) {
```

Comment:
Hehe, I preferred the simpler guard + return clause. No real reason to force rightward shift for such a simple thing.

Not blocking though, this is also fine...